lib/core/src/lib/form/services/date-format-translation.service.spec.ts

```diff
@@ -0,0 +1,72 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TestBed } from '@angular/core/testing';
+import { DateFormatTranslationService } from './date-format-translation.service';
+import { TranslateModule } from '@ngx-translate/core';
+import { CoreTestingModule } from '../../testing/core.testing.module';
+
+describe('DateFormatTranslationService', () => {
+    let service: DateFormatTranslationService;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
```

Comment:
you don't need this block, the service does not use Translation or Root, and it has providedIn: root, so you don't need setting the "providers" section too

Reply:
removed